nixos/infrastructure/vagrant.nix

```diff
@@ -49,7 +49,7 @@ in {
       description = "Vagrant user";
       group = "users";
       extraGroups = [ "login" "service" "docker" ];
-      password = "vagrant";
+      hashedPassword = "$5$xS9kX8R5VNC0g$ZS7QkUYTk/61dUyUgq9r0jLAX1NbiScBT5v1PODz4UC";
```

Comment:
Can you put a comment with the plaintext password here so that the uninitiated know what it is? It's a public password anyway, so ...

Reply:
done